NoClassDefFoundError: org/jline/terminal/impl/MouseSupport when clicking in console

### Expected Behavior

Clicking inside the console (left or right mouse button) should not cause any errors or crash the console thread.

### Actual Behavior

When clicking inside the terminal window while Velocity is running, the console throws a `NoClassDefFoundError` related to `org.jline.terminal.impl.MouseSupport`.
The proxy itself continues running, but the console input stops working.

### Steps to Reproduce

1. Start Velocity normally.
2. Let the server run for some time (the issue does not always occur immediately).
3. Click anywhere inside the console window (either left or right mouse button).
4. Occasionally, the console throws a `NoClassDefFoundError` related to `org.jline.terminal.impl.MouseSupport`.

### Plugin List

velocity, signedvelocity, skinsrestorer, velocityautorestart

### Velocity Version

Velocity 3.4.0-SNAPSHOT-git-ac6d54fc
Copyright 2018-2025 Velocity Contributors. Velocity is licensed under the terms of the GNU General Public License v3.
discord.gg/beer - GitHub
You are running a development build of Velocity.

### Additional Information

```log
[16:02:14 ERROR]: Exception in thread "main" java.lang.NoClassDefFoundError: org/jline/terminal/impl/MouseSupport
[16:02:14 ERROR]: at org.jline.terminal.impl.AbstractTerminal.trackMouse(AbstractTerminal.java:296)
[16:02:14 ERROR]: at org.jline.reader.impl.LineReaderImpl.doCleanup(LineReaderImpl.java:2540)
[16:02:14 ERROR]: at org.jline.reader.impl.LineReaderImpl.cleanup(LineReaderImpl.java:2526)
[16:02:14 ERROR]: at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:727)
[16:02:14 ERROR]: at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:468)
[16:02:14 ERROR]: at net.minecrell.terminalconsole.SimpleTerminalConsole.readCommands(SimpleTerminalConsole.java:158)
[16:02:14 ERROR]: at net.minecrell.terminalconsole.SimpleTerminalConsole.start(SimpleTerminalConsole.java:141)
[16:02:14 ERROR]: at com.velocitypowered.proxy.Velocity.main(Velocity.java:85)
[16:02:14 ERROR]: Caused by: java.lang.ClassNotFoundException: org.jline.terminal.impl.MouseSupport
```

--- a/proxy/src/main/java/com/velocitypowered/proxy/console/VelocityConsole.java
+++ b/proxy/src/main/java/com/velocitypowered/proxy/console/VelocityConsole.java
@@ -135,6 +135,8 @@ public final class VelocityConsole extends SimpleTerminalConsole implements Cons
   protected LineReader buildReader(final LineReaderBuilder builder) {
     return super.buildReader(builder
         .appName("Velocity")
+        // Explicitly disable mouse support on the builder
+        .option(LineReader.Option.MOUSE, false)
         .completer((reader, parsedLine, list) -> {
           try {
             List<String> offers = this.server.getCommandManager()

